src/Utils.h

```diff
@@ -26,4 +26,12 @@
 # define ZeroMemory(ptr,len) memset(ptr, 0, len)
 #endif
 
+#ifdef USE_GRPC
+# include <QDebug>
+# include <string>
+# include <grpcpp/support/string_ref.h>
```

Comment:
On my system this has to be
```
# include <grpc++/support/string_ref.h>
```
instead. Did this name change with some version of grpc?